tests/clientEncryption/clientEncryption-ctor-error-002.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+MongoDB\Driver\ClientEncryption::__construct() with invalid option types
```

Comment:
Shouldn't this be a test for `Manager::createClientEncryption()` since the constructor is disabled? I see this is `clientEncryption-ctor-error-002.phpt`, so perhaps move all of those over to the Manager name.

Reply:
Ah, yes. Those were left over from when I initially built this with a public constructor. I moved them to the manager folder and renamed them accordingly.